ca/README.md

```diff
@@ -0,0 +1,91 @@
+CA
+===
+
+Budowanie obrazu
+----------------
+Obraz api powstał na podstawie oficjalnego obrazu [Pythona][python-url] dostępnego na platformie [DockerHub][python-image]. Do obrazu bazowego dodany został skrypt [EasyRSA][easyrsa-url]. Kontener ten ma za zadanie stworzenie spójnego środowiska do tworzenia certyfikatów z możliwości wykorzystanie narzędzi Docker'a.
+
```

Comment:
Żeby nas recenzenci nie zjedli należy to przedstawić tak że możliwość uruchomienia CA na tym samym hostc-ie co API i automatyczne udostępnianie wystawionych certyfikatów jest przewidziane do prac programistycznych i testów.

Domyślny tryb działania to postawienie CA w docker na oddzielnej maszynie (odłączonej od internetu). Eksport wygenerowanych certyfikatów na pamięć przenośną USB i zaimportowanie w kontenerze API.

Reply:
Czyli w trakcie normalnej, domyslnej pracy kontener z certyfikatami nie bedzie miał żadnego dostępu do api? Uniemożliwia to automatyczne dodawanie informacji o wygenerowanych certyfikatach przy ich tworzeniu (utowrzenie obiektu `Authorization`). Import certyfikatów za pomocą skryptu uruchamianego ręcznie i czytającego volume podłączony do kontenera api?

Reply:
Wydaje mi się że tak. W trakcie importu należałoby równocześnie uwtorzyć odpowiednie wpisy w bazie danych.

Reply:
https://github.com/ktrzcin1/api/blob/master/api/crt_to_json.py
https://github.com/ktrzcin1/api/blob/master/api/auth_import.sh

'crt_to_json.py' parsuje dane z certyfikatu do json-a a auth_import.sh odpala kontener i ładuje je do serializera. Myślałem nad automatycznym tworzeniu pliku .json z danymi przy generowaniu/kopiowaniu certyfikatu z ca, nie trzeba będzie kopiować całego certyfikatu na host-a api. Chyba dobrze by było kod pythonowy ze skryptu 'auth_import.sh' zapakować do kontenera api.